### PR TITLE
feat(cloudfront): retrieve default distribution metrics

### LINKED ISSUE #28893: cloudfront: get default distribution metrics
### Describe the feature

CloudFront distributions have [default metrics](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/viewing-cloudfront-metrics.html) available, but there's a need for metricXyz() that can retrieve these as cloudwatch.Metric objects.

### Use Case

Distributions have both additional metrics and default metrics. 
While all [metric retrieval functions are implemented for additional metrics](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-cloudfront/lib/distribution.ts#L405), it would be very convenient to have similar functions available for default metrics as well.

### Proposed Solution

_No response_

### Other Information

_No response_

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.123.0

### Environment details (OS name and version, etc.)

irrelevant

### PR DESCRIPTION
**Pull request by badmintoncryer**
This PR implements functions to retrieve [default distribution metrics](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/viewing-cloudfront-metrics.html).

```ts
declare const dist: cloudfront.Distribution

const metric = dist.metricRequests()
```

Closes #28893

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts [hunk 1]
```diff
@@ -1294,7 +1294,16 @@ describe('Distribution metrics tests', () => {
...
+  const defaultMetrics = [
+    { name: 'Requests', method: 'metricRequests', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'BytesDownloaded', method: 'metricBytesDownloaded', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'BytesUploaded', method: 'metricBytesUploaded', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'TotalErrorRate', method: 'metricTotalErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: '4xxErrorRate', method: 'metric4xxErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: '5xxErrorRate', method: 'metric5xxErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+  ];
+
+  test.each(additionalMetrics.concat(defaultMetrics))('get %s metric', (metric) => {
```
Comment (GavinZZ) [inline packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts:1306]: Based on comments you added, same of the metrics' default statistic are `@default - sum over 5 minutes`, but the test is expecting the metricObj to `statistic: 'Average'`.

**Review by GavinZZ — COMMENTED** [review]
Love it, a minor question.

File: packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts (hunk 1 above)
Comment (badmintoncryer) [reply packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts:1306]: I made an error in the comment description. I have corrected it to 'default - average over 5 minutes' for all.

**Comment by badmintoncryer** [comment]
@GavinZZ Thank you for your review! I've updated comments. Please confirm it agin.

File: packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts (hunk 1 above)
Comment (GavinZZ) [reply packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts:1306]: According to the `CloudWatchConsoleServiceDirectory.json` file from [this repo](https://github.com/cdklabs/awscdk-service-spec/blob/main/sources/CloudWatchConsoleServiceDirectory/CloudWatchConsoleServiceDirectory.json), it seems that the previous comment were correct. 
```
"metrics": [
          {
            "id": "AWS::CloudFront::Distribution:Requests",
            "name": "Requests",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:TotalErrorRate",
            "name": "TotalErrorRate",
            "defaultStat": "Average"
          },
          {
            "id": "AWS::CloudFront::Distribution:BytesDownloaded",
            "name": "BytesDownloaded",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:BytesUploaded",
            "name": "BytesUploaded",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:4xxErrorRate",
            "name": "4xxErrorRate",
            "defaultStat": "Average"
          },
          {
            "id": "AWS::CloudFront::Distribution:5xxErrorRate",
            "name": "5xxErrorRate",
            "defaultStat": "Average"
          }
        ]
}
```

Comment (GavinZZ) [reply packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts:1306]: I think the correct thing to do here is to set the default statistic like [this](https://github.com/aws/aws-cdk/blob/0129288695c244b1c3ba8523f459669ffb7cd2c1/packages/aws-cdk-lib/aws-lambda/lib/function.ts#L700) and update the tests accordingly.

Comment (badmintoncryer) [reply packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts:1307]: Thank you for your suggestion. I agree with what you have said. 
I have modified some of the metrics default to 'sum'!

**Review by GavinZZ — APPROVED** [review]
LGTM. Thank you for making the changes and contributing!

**Pull request merged**